index.html

```diff
@@ -1,30 +1,29 @@
 <!DOCTYPE html>
-<html lang="ja">
+<html lang="en">
   <head>
-    <meta http-equiv="content-type" charset="UTF-8">
+    <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" type="text/css" />
-    <title>TerraceTechフロントエンドエンジニア養成所（未追跡ファイルに追加されない問題取り組み中）</title>
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
   </head>
   <body>
     <div>
-      <ul id="js-list">
-      </ul>
+      <ul id="js-list"></ul>
     </div>
-    <script>    
-      var ul = document.getElementById('js-list');
-      for(var i=1; i<=2; i++) {
-      var li = document.createElement('li');
-      li.textContent = a$i;
-      ul.appendChild(li);
-      var a = document.createElement('a');
-      a.href= a$i.html;
-      li.appendChild(a);
-      var img = document.createElement('img');
-      img.src = '/img/bookmark.png';
-      a.appendChild(img);
+  <script>   
+    const ul = document.getElementById('js-list');
+    for(let i=1; i<=2; i++) {
+    const li = document.createElement('li');
+    ul.appendChild(li);
+    const a = document.createElement('a');
+    a.textContent = `a${i}`;
+    a.href= `a${i}.html`;
+    li.appendChild(a);
+    const img = document.createElement('img');
+    img.src = '/img/bookmark.png';
```

Comment:
this is answer. 
```js
<ul>
  <li><a href="a1.html"><img src="/img/bookmark.png">a1</a></li>
  <li><a href="a2.html"><img src="/img/message.png">a2</a></li>
</ul>
```

but your code include "bookmark.png" as all image.

Reply:
I'm sorry. I overlooked the src attribute of the image tag here.
I was doing the same thing in assignment 3, so I'm learning.

Reply:
Thank you for your comments.
I apologize for the incorrect name of the img.
We will fix it immediately.